_aggregations/bucket/diversified-sampler.md

```diff
@@ -8,17 +8,19 @@ redirect_from:
   - /query-dsl/aggregations/bucket/diversified-sampler/
 ---
 
-# Diversified sampler aggregations
+# Diversified sampler
 
-The `diversified_sampler` aggregation lets you reduce the bias in the distribution of the sample pool. You can use the `field` setting to control the maximum number of documents collected on any one shard which shares a common value:
+The `diversified_sampler` aggregation lets you reduce the bias in the distribution of the sample pool by de-duplicating documents with the same `field`. It does so by using the `max_docs_per_value` and `field` settings, which limit the maximum number of documents collected on a shard for the provided `field`. The `max_docs_per_value` setting is an optional parameter used to determine up to how many documents per `field` will be returned. The default value of this setting is `1`.
```

Comment:
Last sentence: "used to determine the maximum number of documents that will be returned per `field`"?